tests/icalendar_test.py

```diff
@@ -71,3 +71,10 @@ def test_split_ics_missing_timezone():
     """testing if we detect the missing timezone in splitting"""
     cal = _get_text('event_dt_local_missing_tz')
     split_ics(cal, random_uid=True, default_timezone=LOCALE_BERLIN['default_timezone'])
+
+
+def test_windows_timezone(caplog):
+    """ Test if a windows tz format work"""
```

Comment:
Sorry, one more:
```suggestion
    """Test if a windows tz format works"""
```

Reply:
I forget this one... But I've done it now